Corona-Warn-App/src/main/java/de/rki/coronawarnapp/datadonation/analytics/modules/testresult/AnalyticsTestResultDonor.kt

```diff
@@ -30,12 +30,6 @@ abstract class AnalyticsTestResultDonor(
 ) : DonorModule {
 
     override suspend fun beginDonation(request: DonorModule.Request): DonorModule.Contribution {
-        val scannedAfterConsent = testResultSettings.testScannedAfterConsent.value
-        if (!scannedAfterConsent) {
-            Timber.d("Skipping TestResultMetadata donation (scannedAfterConsent=%s)", scannedAfterConsent)
-            return TestResultMetadataNoContribution
-        }
-
```

Comment:
Why is the check not necessary anymore?

Reply:
I changed the logic to be more like the submission key implementation. The testRegisteredAt is only set when there is consent at registration. That field is checked here anyway, so scannedAfterConsent is not necessary any more.